.pipelines/publish-sku.yaml

```diff
@@ -0,0 +1,39 @@
+trigger: none
+pr: none
+
+jobs:
+- job: publish_sku
+  pool:
+    name: $(BUILD_POOL)
+  steps:
+  - script: |
+      docker run --rm \
+      -v ${PWD}:/go/src/github.com/Azure/aks-engine \
+      -w /go/src/github.com/Azure/aks-engine \
+      ${DEIS_GO_DEV_IMAGE} make bootstrap
+    displayName: Update tools
+  - script: |
+      docker run --rm \
+      -v ${PWD}:/go/src/github.com/Azure/aks-engine \
+      -w /go/src/github.com/Azure/aks-engine \
+      -e AZURE_TENANT_ID=${AZURE_TENANT_ID} \
+      -e AZURE_CLIENT_ID=${AZURE_CLIENT_ID} \
+      -e AZURE_CLIENT_SECRET=${AZURE_CLIENT_SECRET} \
+      -e SKU_PREFIX=${SKU_PREFIX} \
+      -e SKU_TEMPLATE_FILE=${SKU_TEMPLATE_FILE} \
+      -e PUBLISHER=${PUBLISHER} \
+      -e OFFER=${OFFER} \
+      ${DEIS_GO_DEV_IMAGE} ./vhd/publishing/new-windows-sku.sh
+    displayName: Publishing new SKU
```

Comment:
should we avoid the word "publish" since we're not actually hitting the publish button?

Reply:
Sure, do you have any suggestions? i couldn't think of a good term to use :(

Reply:
Create?

Reply:
yeah either create or add. Slight preference for create.

Reply:
updated to create and also specified marketplace sku in a few spots